modules/apps/item-selector/item-selector-test/src/testIntegration/java/com/liferay/item/selector/test/ItemSelectorCriterionHandlerTest.java

```diff
@@ -43,7 +44,8 @@ public void setUp() throws Exception {
 		_bundleContext = _bundle.getBundleContext();
 
 		_serviceReference = _bundleContext.getServiceReference(
-			TestItemSelectorCriterionHandler.class);
+			(Class<BaseItemSelectorCriterionHandler<TestItemSelectorCriterion>>)
```

Comment:
Hey @regiCesar21, we can skip this class because it is not present on the release bundle since it is a test class. Thx